app/actions/adventure.ts

```diff
@@ -0,0 +1,35 @@
+"use server";
+import prisma from "@/lib/db";
+import { TopLevelSpecialCases } from "@/lib/editor/DEPRECATED-editor-options";
+import { auth } from "@clerk/nextjs";
+
+export const resetDevAgent = async (id: string) => {
+  const { userId } = auth();
+  if (!userId) throw new Error("No user");
+
+  const adventure = await prisma.adventure.findUnique({
```

Comment:
Not important but you can use `getAdventure`

Reply:
Yeah but that includes some stuff I don't really care about, like `devAgent` and `NarrativeSnippet`